packages/nameguard-react/src/components/icons/ExternalLinkIcon.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from "react";
+import cc from "classcat";
+import { checkResultCodeTextColor } from "../../utils/text";
+import { CheckResultCode } from "@namehash/nameguard";
+
+export const ExternalLinkIcon = ({
+  isInteractive = false,
+  onClick = () => {},
+  className = "",
+}) => (
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    viewBox="0 0 20 20"
+    fill="none"
+    className={cc([
+      className,
+      "ng-w-[12px] ng-h-[10px] ng-fill-current ng-transition",
+      checkResultCodeTextColor(CheckResultCode.skip, isInteractive),
```

Comment:
It doesn't make sense to me why there should be any reference to the idea of a check result code inside the UI component for an external link icon.